utilities/src/main/java/io/syndesis/qe/utils/DbUtils.java

```diff
@@ -75,4 +103,59 @@ public void deleteRecordsInTable(String tableName) {
 			log.error("Error: " + ex);
 		}
 	}
+
+	public Integer insertInDB(String tableName, List<List<String>> dataTable) {
+
+		String sql = null;
+		switch (tableName.toLowerCase()) {
+			case "todo":
+//				INSERT INTO TODOx(task) VALUES('Joe');
+				sql = String.format("INSERT INTO todo(task) VALUES(?)");
+				break;
+			case "contact":
+//				INSERT INTO CONTACT(first_name, last_name, company, lead_source) VALUES('Josef','Stieranka','Istrochem','db');
+				sql = String.format("INSERT INTO CONTACT(first_name, last_name, company, lead_source, create_date) VALUES(?,?,?,?,now())");
+				break;
+		}
+
+		final PreparedStatement st;
+		try {
+			st = dbConnection.prepareStatement(sql);
+			int i = 1;
+			for (List<String> dataRow : dataTable) {
+				for (String item : dataRow) {
+					st.setString(i, item);
+					i++;
+				}
+			}
+			int newId = st.executeUpdate();
+			log.info("***NEW INT: *{}*", newId);
+			return newId;
+		} catch (SQLException e) {
+			log.error("SQL ERROR: " + e);
+		}
+		return null;
+	}
+
+	public List<Integer> getCompletedByTask(String task) {
```

Comment:
Same as for method above.

Reply:
@dsimansk I have feeling construction of SQL query should be part of DbUtils, WDYT?

Reply:
As I've described in the other comment. DbUtils should be more like entry point how to retrieve desired data for a test, not necessary hide whole validation logic etc. With a retrieved ResultSet you have more flexibility to assert all kind of values when needed.